week4/Project.md

```diff
@@ -0,0 +1,24 @@
+# Build a JS Quiz
+
+Your project for this week is to build a **one-page** JavaScript quiz. It might look similar to the one in the image below, but feel free to be creative.
+
+## These user stories will help you build the app:
+
+* As a player, I want to click on a button to start playing and see the first question.
+* As a player, I want to see a multiple choice question with four possible answers.
+* As a player, I want to choose an answer and to see it highlighted (colored, have its box checked, etc.).
+* As a player, I want to click on a button to submit my answer and know whether it's the right one.
+* As a player, I want to know how many questions I have left (e.g., 'Question 2/5'.
+* As a player, I want to receive one point for each correct answer and see my results when the games ends.
+* As a player, I want to be able ti quit the game and return to the home screen.
```

Comment:
To make it clear that this is a one page app, I'd suggest change this user story slightly:
```suggestion
* As a player, I want to be able to press a button to restart the game
```